src/app/services/configuration-control.service.ts

```diff
@@ -525,7 +525,7 @@ export class ConfigurationControlService {
     // make a copy of matched config items having changed values
     const matchedConfig = defaultConfig.filter(e1 => {
       if (changedConfiguration.hasOwnProperty(e1.key)) {
-        if (e1.type == 'JSON' || e1.type == 'bucket') {
+        if (e1.type == 'JSON') {
```

Comment:
This is the only place we had bucket type check?

Reply:
Yes